packages/transaction-controller/src/TransactionController.ts

```diff
@@ -758,6 +764,7 @@ export class TransactionController extends BaseControllerV1<
       networkClientId,
     };
 
+    await this.#updateTransactionLayer1GasFee(ethQuery, transactionMeta);
```

Comment:
Would it make sense to call this from within `updateGasProperties` so all the gas related values are updated in one place?

Reply:
Relocated layer 1 gas fee updater into `updateGasProperties`